docs/API.Stream.ConsumeNextChunk.md

```diff
@@ -41,26 +43,172 @@ position | Defines the position of the chunk in the stream |  Position ID return
 
 ### Call ###
 
-    curl -vX GET <cm-well-host>/example.org/Individuals/_consume?position=eQA5eJwzNDE1sDQ3NDI0tjQ2rjGoMTE0MDI1NjS0BIEa_dSKxNyCnFS9_KJ0fc-8lMyyzJTSxJzimpoa&format=json
+    curl -vX GET <cm-well-host>/example/Geography/_consume?position=eQA5eJwzNDE1sDQ3NDI0tjQ2rjGoMTE0MDI1NjS0BIEa_dSKxNyCnFS9_KJ0fc-8lMyyzJTSxJzimpoa&format=json
 
 ### Results ###
 
+The following results are truncated.
+
+    < HTTP/1.1 200 OK
+    < X-CMWELL-BG: O
+    < X-CMWELL-RT: 546ms
+    < X-CM-WELL-N-LEFT: 162130
+    < X-CMWELL-Version: 1.5.51
+    < X-CMWELL-Hostname: c649taycsnd19.int.data.com
```

Comment:
can we change hostname header to `cm-well-host` or `127.0.0.1` or something else of sort?